src/main/java/FrameworksDrivers/ScheduleDrawing.java

```diff
@@ -0,0 +1,179 @@
+package FrameworksDrivers;
+
+import Entities.Event;
+import Entities.Schedule;
```

Comment:
clean architecture violation: can't be importing entities because that's skipping through layers, may have to make another use case class